devel/libical/files/patch-nodocs.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 23898b6..fb7e452 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -347,7 +347,7 @@ endif()
+ ################# build subdirs ########################
+ 
+ add_subdirectory(design-data)
+-add_subdirectory(doc)
++# add_subdirectory(doc)
```

Comment:
Not saying that you should wait for upstream to resolve the issue, but if you have the energy to come up with a proper patch or at least submit a feature request upstream that implements a configure option to switch building the docs on and off, that would be helpful.